roles/download/defaults/main/main.yml

```diff
@@ -114,13 +114,13 @@ typha_enabled: false
 calico_apiserver_enabled: false
 
 flannel_version: "v0.22.0"
-flannel_cni_version: "v1.1.2"
+flannel_cni_version: "v1.2.0"
```

Comment:
This was intentionally downgraded somewhat recently see this https://github.com/kubernetes-sigs/kubespray/pull/10205#issuecomment-1583451821

Reply:
Got it, I'll change it and fix the `kubelet_checksums` versions